src/handlers.js

```diff
@@ -1,15 +1,28 @@
+import { gameTools } from "./store.js";
 import  { getSolutions }  from "./winningStates.js";
 
 const createBoard = (size) => {
   const gameContainer = document.querySelector(".game");
-  gameContainer.classList.remove("hidden");
   gameContainer.innerHTML = "";
   for (let i = 0; i < size ** 2; i++) {
     const cell = document.createElement("div");
     cell.classList.add(`grid-cell`);
     cell.dataset.value = i;
     gameContainer.appendChild(cell);
   }
+  const style = document.querySelector(".gridStyle");
```

Comment:
why is that? can't you use css for this (e.g. using css variable)? pushing style element directly to the dom doesn't look like a good idea

Reply:
as it seemed to me the nth:child selector was the best way for designing the grid  - but i realized I cant use a variable on the selector - .grid-cell:nth-child(var(--boardSize)n) didnt work. 
so I tried to find a solution that will  let me control the selector with variable and this is what i came up with..